### DISCUSSION TITLE
How do Imports work in Odin?

### DISCUSSION THREAD

**Discussion by DreepyYunky** [odin-lang/Odin#2665] [Q&A] [answered]
I have a question, Can I create a prefix in odin (like `core:` or `vendor:`). If I want want to import a file, can I just use the file's name?

**Comment by programandala-net** [comment, marked as answer]
Yunky86 escribió/skribis/wrote/scrit (2023-07-18T09:12:56-0700):

> Can I create a prefix in odin (like `core:` or `vendor:`)

Yes, using the `-collection:` option of the `build` command. See the usage,
extracted from `odin help build`:


```
-collection:<name>=<filepath>
        Defines a library collection used for imports
        Example: -collection:shared=dir/to/shared
        Usage in Code:
                import "shared:foo"
```


-- 
Marcos Cruz
http://programandala.net

**Comment by Kelimion** [comment]
> I have a question, Can I create a prefix in odin (like `core:` or `vendor:`). If I want want to import a file, can I just use the file's name?

Yes, using the `-collection:name=dir` flag, after which you can `import "name:foo"`.
